workflow-templates/im-deploy-az-app-manually.yml

```diff
@@ -103,27 +110,71 @@ jobs:
           scope: ${{ env.ENVIRONMENT }}
           create-output-variables: true
         env:
-          # This variable is used to swap slots and annotate app insights
+          # This variable is used to swap slots
           RESOURCE_GROUP@dev: ''
           RESOURCE_GROUP@qa: ''
           RESOURCE_GROUP@stage: ''
+          RESOURCE_GROUP@stage-secondary: ''
           RESOURCE_GROUP@demo: ''
           RESOURCE_GROUP@uat: ''
           RESOURCE_GROUP@prod: ''
+          RESOURCE_GROUP@prod-secondary: ''
+
+          # Primary resource group used to pull azure credentials and secrets from
+          PRIMARY_RESOURCE_GROUP@dev: ''
+          PRIMARY_RESOURCE_GROUP@qa: ''
+          PRIMARY_RESOURCE_GROUP@stage stage-secondary: ''
+          PRIMARY_RESOURCE_GROUP@demo: ''
+          PRIMARY_RESOURCE_GROUP@prod prod-secondary: ''
+
           # This variable is used to deploy the app, swap slots, annotate app insights and send a teams notification
           AZ_APP_NAME@dev: ''
           AZ_APP_NAME@qa: ''
           AZ_APP_NAME@stage: ''
+          AZ_APP_NAME@stage-secondary: ''
           AZ_APP_NAME@demo: ''
           AZ_APP_NAME@uat: ''
           AZ_APP_NAME@prod: ''
+          AZ_APP_NAME@prod-secondary: ''
+          # TODO: Uncomment if you don't use an MSI with your app service
+          # This variable is used to assign a managed identity on slot creation
+          # AZ_APP_MSI@dev: ''
+          # AZ_APP_MSI@qa: ''
+          # AZ_APP_MSI@stage stage-secondary: ''
+          # AZ_APP_MSI@demo: ''
+          # AZ_APP_MSI@uat: ''
+          # AZ_APP_MSI@prod prod-secondary: ''
           # This variable is used to annotate app insights
           APP_INSIGHTS_NAME@dev: ''
           APP_INSIGHTS_NAME@qa: ''
-          APP_INSIGHTS_NAME@stage: ''
+          APP_INSIGHTS_NAME@stage stage-secondary: '' # Currnetly appinsights isn't available in West Central US and you have to use West US 2 AppInsights
           APP_INSIGHTS_NAME@demo: ''
           APP_INSIGHTS_NAME@uat: ''
-          APP_INSIGHTS_NAME@prod: ''
+          APP_INSIGHTS_NAME@prod prod-secondary: '' # Currnetly appinsights isn't available in West Central US and you have to use West US 2 AppInsights
+          # Variables used for referencing secrets
+          PRIMARY_ENVIRONMENT@dev qa stage prod demo: '${{ github.event.inputs.environment }}'
```

Comment:
This variable name confused me at first. Maybe it would be better if it were something like `ENVIRONMENT_FOR_GITHUB_SECRETS`

Reply:
Updated to GITHUB_SECRETS_ENVIRONMENT